client/packages/common/src/intl/locales/fr/common.json

```diff
@@ -1660,6 +1740,8 @@
   "messages.only-closed-periods-visible": "Note : seules les périodes terminées sont visibles",
   "messages.outbound-shipment-created-on": "Livraison sortante créée le {{date}}",
   "messages.over-allocated": "En raison des tailles d'emballage disponibles, une quantité totale de {{quantity}} a été allouée au lieu de {{issueQuantity}}",
+  "messages.over-allocated-line": "En raison de la taille de l'emballage, une quantité de {{quantité}} a été attribuée au lieu de {{quantité d'émission}}.",
```

Comment:
This kind of thing will be a problem, the {{}} stuff shouldn't be changed, can possibly work around that in our code, or by providing context to DeepL?